Added Binary Tree Preorder and Postorder Traversal Code

RgnDunes opened the pull request:
**IMPORTANT: Please do not create a Pull Request without an issue.**

### All Submissions:

* [x] I have read the CONTRIBUTING document.
* [x] My code is written in **Python3** and is ending with _.py_
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] I have checked that my submission does pass the test on LeetCode.com 
* [x] Does your filename follow the naming Conventions?
* [x] Have you linked your PR to an Issue?

**Closing issues**

closes #723

riwim reviewed (changes requested):
Thanks for your contribution ✨

Please
1. Read CONTRIBUTING.md
1. Submit **only one solution**
1. Do not change `PULL_REQUEST_TEMPLATE.md`
1. Remove backticks around `closes #XXXX` in your comment (to properly link your opened issue)

RgnDunes commented:
> Thanks for your contribution ✨
> 
> Please
> 
> 1. Read CONTRIBUTING.md
> 2. Submit **only one solution**
> 3. Do not change `PULL_REQUEST_TEMPLATE.md`
> 4. Remove backticks around `closes #XXXX` in your comment (to properly link your opened issue)

Made all the necessary changes you asked for.
> 1. Read
> 2. Deleted one solution so adding just one solution.
> 3. Undone all the changes made in `PULL_REQUEST_TEMPLATE.md`
> 4. Removed backticks around `closes #XXXX` in my comment (to properly link your opened issue)

riwim commented on PULL_REQUEST_TEMPLATE.md:
```diff
@@ -2,12 +2,12 @@
-* [ ] I have checked that my submission does pass the test on LeetCode.com 
-* [ ] Does your filename follow the naming Conventions?
-* [ ] Have you linked your PR to an Issue?
+- [ ] I have read the CONTRIBUTING document.
+- [ ] My code is written in **Python3** and is ending with _.py_
+- [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+- [ ] I have checked that my submission does pass the test on LeetCode.com
+- [ ] Does your filename follow the naming Conventions?
+- [ ] Have you linked your PR to an Issue?
```
1., 2. and 4. seems good.
You still didn't reset the template to its original state.
```suggestion
* [ ] I have read the CONTRIBUTING document.
* [ ] My code is written in **Python3** and is ending with _.py_
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ ] I have checked that my submission does pass the test on LeetCode.com 
* [ ] Does your filename follow the naming Conventions?
* [ ] Have you linked your PR to an Issue?
```


Copy&paste the original content or (my recommendation) read about the several possibilities to undo changes in Git:
[Undoing Things](https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things)

RgnDunes replied:
Sir I have copied and pasted original content from original PULL_REQUEST_TEMPLATE.md file and commited it.

riwim reviewed (changes requested):
I put some suggested changed in a comment.

riwim commented:
Good work 🎉

vJechsmayr reviewed (approved):
Thanks for contributing - Merged!

Runtime: 28 ms

Happy Hacktoberfest!
